extensions/sql-migration/src/api/azure.ts

```diff
@@ -124,7 +124,7 @@ export type SqlVMServer = {
 };
 export async function getAvailableSqlVMs(account: azdata.Account, subscription: Subscription, resourceGroup: azureResource.AzureResourceResourceGroup): Promise<SqlVMServer[]> {
 	const api = await getAzureCoreAPI();
-	const path = `/subscriptions/${subscription.id}/resourceGroups/${resourceGroup.name}/providers/Microsoft.SqlVirtualMachine/sqlVirtualMachines?api-version=2017-03-01-preview`;
+	const path = encodeURI(`/subscriptions/${subscription.id}/resourceGroups/${resourceGroup.name}/providers/Microsoft.SqlVirtualMachine/sqlVirtualMachines?api-version=2017-03-01-preview`);
```

Comment:
@aasimkhan30 We should look into doing this in azurecore instead - this isn't something specific to sql-migration unless I'm missing something. Doing it for all Azure REST calls seems reasonable.

Reply:
I agree. I will add that to azurecore. However, we still need this piece of code in the extension as it won't be available before next release.